Source/WebKit/UIProcess/Extensions/WebExtensionController.h

```diff
@@ -56,6 +56,7 @@ namespace WebKit {
 
 class ContextMenuContextData;
 class WebExtensionContext;
+class WebInspectorUIProxy;
```

Comment:
The forward class declaration can be conditionalized on `ENABLE(INSPECTOR_EXTENSIONS)`.